src/controllers/Poll.js

```diff
@@ -63,8 +64,9 @@ async function handleSelector(metaResources, razeedashSender, selector, formatte
         } else if (r.statusCode === 403 || r.statusCode === 404 || r.statusCode === 405) {
           // 403 Forbidden, 404 NotFound, 405 MethodNotAllowed
           // These statusCodes are expected. Processing should continue.
-          log.debug(`Could not get resource '${r['resource-metadata'].uri()}' : ${JSON.stringify(r.error)}`);
-        } else { // Unexpected status code. Error out of this flow.
+          const uri = `${r['resource-metadata']._path}/${r['resource-metadata'].kind}`;
```

Comment:
do you know where `r['resource-metadata']` comes from? can we use your new getter instead? Are they even the same thing?

also i think you should be using `.path` instead of `._path`